docker/de/Readme.md

```diff
@@ -46,6 +40,22 @@ $ docker stop chaincore
 
 ## Other features
 
+#### Persist data outside of Docker
+
+Chain Core stores data in three locations: a data directory, a separate Postgres database, and a log directory. To persist the data outside the container, create directories on your development machine and mount them to the container on `docker run`:
+
+```sh
+$ mkdir -p /path/to/store/datadir
+$ mkdir -p /path/to/store/postgres
+$ mkdir -p path/to/store/logs
+$ docker run -p 1999:1999 \
```

Comment:
@boymanjor Would you prefer to see name in the getting started docs too? We've never included it in our main guide.